common/models/job-monitor.js

```diff
@@ -6,72 +6,22 @@ const _ = require('lodash'),
 
 module.exports = function(JobMonitor) {
 
```

Comment:
@justinlueders we didn't really maintain this feature after the initial attempt. it originally cleared related postsclusters. but then we added aggregatecluster and event model, and we didn't keep up in the code here. so i thought, better to remove than to have it in incomplete state. afaik we all just wipe the db between system runs now anyway.